docs/sprint-development-plan/Product_Backlog_V2.md

```
@@ -0,0 +1,485 @@
+    > Desc: Como administrador quiero saber cuales son los eventos que más le interesan a mis residentes a través de datos cuantitativos.
+
+### Módulo: Analíticas 
+
+* **[Admin/Personal] Generación de analíticas de habitaciones, incidencias, objetos y espacios.** (`NX-S3.35`)
+    > - Añadir a barra lateral.
+    > - Poder exportar reportes en pdf, incluso reportes excel mas concretos (usando filtros etc...)
+
+
+# Reordenar 
+
+### Módulo: Multi-sede 
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)